cmake/macros/testWrapper.py

```diff
@@ -244,7 +260,20 @@ def _copyTree(src, dest):
         if os.path.isdir(s):
             shutil.copytree(s, d)
         else:
-            shutil.copy2(s, d) 
+            shutil.copy2(s, d)
+
+def _copyFailedDiffFiles(failuresDir, baselineFile, resultFile):
+    baselineName = _addFilenameSuffix(os.path.basename(baselineFile),
+                                      ".baseline")
+    resultName = _addFilenameSuffix(os.path.basename(resultFile), ".result")
+    baselineOutpath = os.path.join(failuresDir, baselineName)
+    resultOutpath = os.path.join(failuresDir, resultName)
+
+    if not os.path.isdir(failuresDir):
+        os.makedirs(failuresDir)
+
+    shutil.copy(baselineFile, baselineOutpath)
+    shutil.copy(resultFile, resultOutpath)
```

Comment:
Sunya noticed that it WBN to print out some kind of message here saying where the diff results are being copied to. That way someone looking at the test output can see the exact path they need to go to to look for more info.

Reply:
Absolutely, will do!

Reply:
Added:
https://github.com/pmolodo/USD/commit/c24838f63735082c515e7dc8b99032397c949c72#diff-20f6c1250283d4110c2cacd19f922472ce55350c2e19d29f9491db6b2e792793R279